### PR TITLE
fix(frontend): remove alt avatar

### PR DESCRIPTION
**Pull request by Haskaris**
Issue ref #6145 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
~~- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)~~ Not relevant
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

**Review by Millefeuille42 — APPROVED** [review]

File: frontend_lib/src/component/Avatar/Avatar.jsx [hunk 1]
```diff
@@ -37,7 +37,6 @@ export class Avatar extends React.Component {
         <img
           className='avatar'
           src={`${avatarBaseUrl}/preview/jpg/${sizeAsNumber}x${sizeAsNumber}/${filenameInUrl}`}
-          alt={props.t('Avatar of {{publicName}}', { publicName })}
```
Comment (Skylsmoi) [inline frontend_lib/src/component/Avatar/Avatar.jsx:40]: I'm expecting this change comes from our recent discussion about the alt attribute.

After writing a bit of guideline in this comment, then thinking back about it, I decided to ctrl+t a bit about "alt".
W3C says its required. MDN says it isn't required but "incredibly useful for accessibility".
Which means I was wrong, it indeed is useful for accessibility.

So... maybe "Avatar of {{publicName}}" is actually the best

**Review by Skylsmoi — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/Avatar/Avatar.jsx (hunk 1 above)
Comment (Haskaris) [reply frontend_lib/src/component/Avatar/Avatar.jsx:40]: Well, I've read this while I was removing the alt... I feel like the Avatar displayed here is purely decorative. Like we know that the line where there will be the avatar, there will be the user name too... 
Or am I wrong?

Looks like in any case, the `alt` should be there, but the value should be empty.

Comment (Skylsmoi) [reply frontend_lib/src/component/Avatar/Avatar.jsx:40]: I agree, `alt=""` is the best here.
I found a good article aboud alt: https://webaim.org/techniques/alttext/

Comment (Haskaris) [reply frontend_lib/src/component/Avatar/Avatar.jsx:40]: Solved in [41219fb](https://github.com/tracim/tracim/pull/6146/commits/41219fbb58e78742229acde6c48d16ae0f65bafa)

**Pull request merged**